cross-dom-bridge-erc20/index.js

```diff
@@ -47,68 +72,33 @@ const setup = async() => {
       l1SignerOrProvider: l1Signer,
       l2SignerOrProvider: l2Signer
   })
-//  l1ERC20 = new ethers.Contract(daiAddrs.l1Addr, erc20ABI, l1Signer)
-//  l2ERC20 = new ethers.Contract(daiAddrs.l2Addr, erc20ABI, l2Signer)
+  l1ERC20 = new ethers.Contract(erc20Addrs.l1Addr, erc20ABI, l1Signer)
+  l2ERC20 = new ethers.Contract(erc20Addrs.l2Addr, erc20ABI, l2Signer)
 }    // setup
 
 
 
-// The ABI fragment for an ERC20 we need to get a user's balance.
-const erc20ABI = [
-    // balanceOf
-    {
-      constant: true,
-      inputs: [{ name: "_owner", type: "address" }],
-      name: "balanceOf",
-      outputs: [{ name: "balance", type: "uint256" }],
-      type: "function",
-    },
-  ]    // erc20ABI
-
-
-const gwei = 1000000000n
-const eth = gwei * gwei   // 10^18
-const centieth = eth/100n
-const dai = eth
-
-
-const reportBalances = async () => {
-  const l1Balance = (await crossChainMessenger.l1Signer.getBalance()).toString().slice(0,-9)
-  const l2Balance = (await crossChainMessenger.l2Signer.getBalance()).toString().slice(0,-9)
-
-  console.log(`On L1:${l1Balance} Gwei    On L2:${l2Balance} Gwei`)
-}    // reportBalances
-
-
-
 const reportERC20Balances = async () => {
   const l1Balance = (await l1ERC20.balanceOf(addr)).toString().slice(0,-18)
   const l2Balance = (await l2ERC20.balanceOf(addr)).toString().slice(0,-18)
-  console.log(`DAI on L1:${l1Balance}     DAI on L2:${l2Balance}`)
+  console.log(`OUTb on L1:${l1Balance}     OUTb on L2:${l2Balance}`)
+
+  if (l1Balance != 0)
```

Comment:
uhh, not sure what is up with the formatting here. Please add explicit brackets for the if statement.